src/content-scanner/ContentScanner.ts

```diff
@@ -123,6 +123,7 @@ export class ContentScanner {
     public static get instance(): ContentScanner {
         if (!ContentScanner.internalInstance) {
             ContentScanner.internalInstance = new ContentScanner(MatrixClientPeg.get().getHomeserverUrl());
+            ContentScanner.internalInstance = new ContentScanner("http://localhost:9000");
```

Comment:
accidental commit ?

Reply:
Yes, of course. This is sub-optimal. I have to change this every time I am developing locally. Maybe I re-add the URL option for the content scanner. This way I can override it locally. What do you think?

Reply:
hmm yes that sucks.
Re-add the `content_scanner.url`, and if it's absent, default to `homeserverUrl` ? That way we have prod config by default, and no possibility of running without contentscanner, that sounds good.

Reply:
Done